specification/cdn/resource-manager/Microsoft.Cdn/preview/2025-01-01-preview/examples/AFDCustomDomains_Create.json

```diff
@@ -0,0 +1,213 @@
+{
+  "parameters": {
+    "api-version": "2025-01-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "customDomainName": "domain1",
+    "customDomain": {
+      "properties": {
+        "hostName": "www.someDomain.net",
+        "tlsSettings": {
+          "certificateType": "ManagedCertificate",
+          "cipherSuiteSetType": "Customized",
+          "minimumTlsVersion": "TLS12",
+          "customizedCipherSuiteSet": {
+            "cipherSuiteSetForTls10": [],
+            "cipherSuiteSetForTls12": [
+              "ECDHE_RSA_AES128_GCM_SHA256"
+            ],
+            "cipherSuiteSetForTls13": [
+              "TLS_AES_128_GCM_SHA256",
+              "TLS_AES_256_GCM_SHA384"
+            ]
+          }
+        },
+        "mtlsSettings": {
+          "secrets": [
+            {
+              "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/secrets/name1"
+            },
+            {
+              "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/secrets/name2"
+            }
+          ],
+          "otherAllowedFqdn": [
+            "foo.contoso.com"
+          ]
+        },
+        "azureDnsZone": {
+          "id": ""
```

Comment:
I believe this bug in your example (missing value for `id`) is causing ModelValidation to crash:

```
ErrorMessage:Cannot read properties of null (reading 'id')
```

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4602971&view=logs&j=14c5c170-9612-5b44-a650-13d3ca524cf8&t=a1928b1b-77e4-58b6-0247-4cedb38901b4&l=183

Reply:
@mikeharder Thanks for taking a look! I went through and corrected all the 2025-01-01-preview/examples/AFDCustomDomains_*.json examples (commit [`87f1a7c652`](https://github.com/Azure/azure-rest-api-specs/pull/32875/commits/87f1a7c6525c9bc26758b72462b642e190d28f6e). The same ModelValidation error is still showing up though. I checked over the rest of 2025-01-01-preview/examples for `"id": ""`, but didn't get more hits. Any ideas what else could be causing the error?